addons/mail/tests/test_mail_activity_plan.py

```diff
@@ -0,0 +1,30 @@
+from odoo.addons.mail.tests.test_mail_activity import ActivityScheduleCase
+from odoo.tests import tagged
+
+
+@tagged('mail_activity', 'mail_activity_plan')
+class TestActivityPlan(ActivityScheduleCase):
```

Comment:
```suggestion
class TestActivityPlan(TransactionCase):
```

Reply:
Why not adding a quick test in "TestActivitySchedule" in test_mail/test_mail_activity_plan ? That way we don't have to define yet another setup and file.

Reply:
I applied changes, thank you for suggestions :) Does it look ok now?